notebooks/api_examples/models/AzureOpenAI.ipynb

```diff
@@ -11,6 +11,21 @@
     "For this, you will provide a few pieces of information from the Azure AI playground:"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6b887dab",
+   "metadata": {
+    "tags": [
+     "parameters",
+     "remove-cell"
```

Comment:
This is not being respected, unfortunately:
https://jupyterbook.org/en/stable/interactive/hiding.html